src/test/java/com/rultor/agents/daemons/EndsDaemonITCase.java

```diff
@@ -103,6 +106,37 @@ public void readsExitCodeCorrectly() throws IOException {
     }
 
     /**
+     * EndsDaemon can deprecateDefaultImage.
+     * @throws IOException In case of error.
+     * @todo @todo #754:30min Implement a deprecation message at the start
+     *  of the process if the project is using the default image
+     *  'yegor256/rultor'.
+     */
+    @Test
+    @Ignore
+    public void deprecateDefaultImage() throws IOException {
+        Assume.assumeFalse(SystemUtils.IS_OS_WINDOWS);
```

Comment:
@original-brownbear do we need such assumptions? Isn't Rultor meant to be run on just on Linux/OS X?